src/Pos/Txp/DB/Balances.hs

```diff
@@ -67,6 +75,25 @@ instance RocksBatchOp BalancesOp where
         else [Rocks.Put (ftsStakeKey ad) (encodeStrict c)]
 
 ----------------------------------------------------------------------------
+-- Overloaded getters (for fixed balances for bootstrap era)
+----------------------------------------------------------------------------
+
+-- TODO: provide actual implementation after corresponding
+-- flag is actually stored in the DB
+isBootstrapEra :: MonadDB m => m Bool
+isBootstrapEra = pure $ not Const.isDevelopment && True
+
+getTotalFtsStake :: MonadDB m => m Coin
+getTotalFtsStake = ifM isBootstrapEra
+    (pure $ foldl' unsafeAddCoin (mkCoin 0) genesisBalances)
```

Comment:
1. AFAIU, now it will be calculated every time. I think it makes sense to do it only once.
2. I think `unsafeIntegerToCoin . sumCoins` is more understandable than `foldl' unsafeAddCoin (mkCoin 0)`.